docu/email-notifications.md

```diff
@@ -2,21 +2,21 @@
 
 See an [implementation example](../examples/using-email-channel/app.php)
 
-## Parameters
-
-Apart of the [mandatory parameters](README.md), you will need:
-
-#### MAILER_HOST
+This project is using the `symfony/mailer` which provides a really good flexibility in order
+to use a [3rd party transport](https://symfony.com/doc/current/mailer.html#using-a-3rd-party-transport).
 
-Value example: `smtp.gmail.com`
+3rd parties such as:
 
-#### MAILER_PORT
+- Amazon SES -> `composer require symfony/amazon-mailer`
+- Gmail -> `composer require symfony/google-mailer`
```

Comment:
Actually, I added this one particularly (to `require-dev`) for the `examples/using-email-channel` :)